README.md

```diff
@@ -191,22 +191,41 @@ py.test
 #### The test data subset
 If you add new tables to the data, you'll need to generate a new subset for testing. We use this nifty subsetting tool: [rdbms-subsetter](https://github.com/18F/rdbms-subsetter).
 
-To build a new test subset, use the `build_test` invoke task:
+To build a new test subset, first create a new database for yourself:
+
+```
+createdb cfdm_new_subset
```

Comment:
I was talking about the way it is currently described:
```
invoke build_test <source> <dest>
invoke dump <source> data/subset.dump
```
It works fine the first time and if you running the build test again, you can delete and recreate the db first.